Documentation/Groups/Groups.md

```diff
@@ -0,0 +1,105 @@
+Note: at the current time, a group is treated as a new "scope" in the workbook. Any parameters created in the group are only visible *inside the group*. This is also true for things like merge, they can only see data inside their group or at the parent level.
+
+## Group types
+The workbook "group" item allows you to add a group of items to a workbook. As the author of a workbook, you specify which type of group it will be. There are 2 types of groups:
+
+* **editable** - the group in the workbook allows you to add/remove/edit the contents of the items in the group. this is most commonly used for layout and visibility purposes.
+
+* **from template** - the group in the workbook loads from the contents of another template by its id. the content of that template is loaded and merged into the workbook at runtime. In edit mode, you cannot modify any of the contents of the group, as they will just load again from the template next time the item loads.
+
+## Load types
+There are several different ways that the content of a group may be loaded. As the author of a workbook, you get to specify when and how the contents of the group will load
+
+* **lazy** (the default) - the group will only load when the item is visible. this allows a group to be used by tab items, and if the tab is never selected, the group never becomes visible, so the content isn't loaded. 
+    - For groups created from a template, the content of the template is not retrieved and the items in the group are not created until the group becomes visible. the user will see progress spinners for the whole group while the content is retrieved.
+    - for editable groups, the content is loaded, but the items that would be displayed in the group are not created until the item becomes visible
+
+* **explicit** - in this mode, a button is displayed where the group would be, and no content is retrieved or created until the user explicitly clicks the button to load the content. This is useful in "show more" scenarios, where the content might be expensive to compute or rarely used. The author can specify the text to appear on the button.
+
+    ##### explicit load settings, showing a configured "Load more" button :
+
+    ![explicit load settings](../Images/groups-explicitly-loaded.png)
+
+    ##### the group before being loaded in the workbook:
+
+    ![explicit load before](../Images/groups-explicitly-loaded-before.png)
+
+    ##### the group after being loaded in the workbook:
+
+    ![explicit load after](../Images/groups-explicitly-loaded-after.png)
+
+* **always** - in this mode, the content of the group is always loaded and created as soon as the workbook loads. This mode is most freqently used when using a group only for layout purposes, where the content will always be visible.
+
+## using templates inside a group
+When a group is configured to load from a template, that by default, that content will only be loaded "lazily", in that it will only load when the group is visible. 
```

Comment:
When a group is configured to load from a template, that content will be loaded "lazily" by default, in that it will only load when the group is visible.